Cargo.toml

```diff
@@ -12,7 +12,7 @@ rust-version = "1.61.0"
 
 [dependencies]
 anyhow = "1.0"
-c2pa = { version = "0.3.0", features = ["file_io"] }
+c2pa = { version = "0.4", features = ["file_io"] }
```

Comment:
This change may or may not make sense. With the less-specific reference, dependabot will be allowed to generate PRs whenever a new c2pa crate is enabled.

Reply:
Let's give it a try,